Added logs_collector_test documentation (#153)
vtex/reliability-research: metrics/logs_collector/README.md

| type | Type of garbage collect | String  |
| stopped_threads_time | Duration where threads spent stopped | milliseconds |
| timestamp | Timestamp in epoch of the start of a Garbage Collection | milliseconds |

## Logs Collector Tests

## Usage
To run the Logs Collector Test, use the following command template:\

> `python3 logs_collector_test.py`

Reviewer: ```suggestion
```shell
$ python3 logs_collector_test.py
```
```

I prefer this type of display for this sort of command. Could you update to it if you think that's better?

##Test performed

> There are 4 input files, each corresponding to a fragments of a Solr log, these files will serve as input for logs_collector, which in the test would run the treatment of the files and generate a file with the result. We have a file with the standard output with the golden file output, which we will use as a basis to compare with the generated output, if they are the same it means that the data treatment is happening correctly otherwise it indicates that there is an error in the data treatment.

Reviewer: ```suggestion
There are 4 input files, each corresponding to fragments of a Solr log. These files will serve as input for logs_collector, which in the test will run their treatment and generate a file with the result. We have a golden file with the expected output, which we use to compare with the output generated in the tests. If they are the same it means the data treatment is happening correctly, otherwise, it indicates that there is an error in the data treatment.
```

That you think? Sound better?
